pkg/crd/crds.go

```diff
@@ -43,7 +44,10 @@ import (
 )
 
 // CRDs to apply
-var crdNames = map[string]bool{"placements.edge.kubestellar.io": true}
+var crdNames = []string{
+	"placementdecisions.edge.kubestellar.io",
+	"placements.edge.kubestellar.io",
+}
```

Comment:
Why not do the set construction once here rather than repeatedly inside `filterCRDsByNames`?

```suggestion
var crdNames = sets.New(
	"placementdecisions.edge.kubestellar.io",
	"placements.edge.kubestellar.io",
)
```